imp_services/displayLogs.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Created by Trevor on 7/3/2015.
+ */
+var Q = require("q");
+
+/* var a = {};
+ a["key1"] = "value1";
+ a["key2"] = "value2";
+ */
+
+var LogTypeMap = {};
```

Comment:
Lets keep the function map and the log type map in one object, An entry should look like this:
LogMap[100] = { type: "Added Pile", callFunction: toStringDefault }

or something like that. Having two objects each with the same key set is less maintainable than having one object.